bundle/manifests/maintenance-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
+  creationTimestamp: null
   labels:
-    control-plane: controller-manager
-    app.kubernetes.io/name: service
-    app.kubernetes.io/instance: controller-manager-metrics-service
     app.kubernetes.io/component: kube-rbac-proxy
```

Comment:
Is this file used?
Still see here kube-rbac-proxy

Reply:
These were leftovers, regenerated the bundle